### PR TITLE
Transactional edits

### PR DESCRIPTION
**Pull request by annatraussnig**
Shiny news texts for the 5-years campaign.

### CODE REVIEW

File: packages/backend-modules/mail/templates/future_campaign_sender_reward.html [hunk 1]
```diff
@@ -67,10 +74,10 @@
                     </p>
                     <p>Merci für Ihre Zeit und Ihre Unterstützung!</p>
```
Comment (fuenkchen) [inline packages/backend-modules/mail/templates/future_campaign_sender_reward.html:75]:
```suggestion
                    <p>Merci für Ihre Zeit, Ihre Hilfe, Ihre Überzeugungskraft. </p>
```

File: packages/backend-modules/mail/templates/future_campaign_sender_reward.html [hunk 2]
```diff
@@ -81,68 +88,90 @@
...
+                      Was im Fall der Republik heisst: Sie haben noch
+                      {{count_open_slots}} Zugänge zu vergeben.
                     </p>
                     <p>
                       Vielleicht wäre das etwas für Ihren Coiffeur, Ihre
-                      Skilehrerin oder Ihren Hausarzt?
+                      Skilehrerin, Ihre Yoga- oder Pokerrunde, Ihren Hausarzt?
                     </p>
                     <p>
-                      <a href="{{link_sender_page}}">Finden Sie es heraus</a>!
```
Comment (fuenkchen) [inline packages/backend-modules/mail/templates/future_campaign_sender_reward.html:92]: Link should not be in a separate paragraph. Should stay the same as before. (in the old version it is the very same)

**Review by fuenkchen — CHANGES_REQUESTED** [review]
Some bits and pieces were missing, just commented on the html file.

**Review by fuenkchen — APPROVED** [review]

**Comment by patrickvenetz** [comment]
@annatraussnig @fuenkchen Thanks tuusig.

**Pull request merged**